src/PackageManagement.VisualStudio/ScriptPackageFile.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Runtime.Versioning;
+using NuGet.Frameworks;
+
+namespace NuGet.PackageManagement.VisualStudio
+{
+    public class ScriptPackageFile : IScriptPackageFile
+    {
+        private readonly NuGetFramework _targetFramework;
+
+        public ScriptPackageFile(string path, NuGetFramework targetFramework)
+        {
+            Path = path.Replace(System.IO.Path.AltDirectorySeparatorChar, System.IO.Path.DirectorySeparatorChar);
+            _targetFramework = targetFramework;
+        }
+
+        public string Path
```

Comment:
Good point, no it is not ok. So just fix the using, and comment on the Path that it is a public API used by init.ps1/install.ps1 users